internal/core/data/event.go

```diff
@@ -134,6 +134,9 @@ func addNewEvent(
 
 	// Add the event and readings to the database
 	if configuration.Writable.PersistData {
+		if e.Created == 0 {
+			e.Created = db.MakeTimestamp()
```

Comment:
Also need to remove setting it in the DB layer for Redis (so saved value is the value set here). Mongo code is going to be removed, so you can ignore it.